couchbase-ansible/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "itemDisplayName": "Advanced Linux : Deploy a Multi VM Couchbase Cluster with Ansible",
```

Comment:
Suggested name: Couchbase cluster with Ansible so that it displays better in the Azure.com index title and the text is not cut off.